client/src/App.js

```diff
@@ -1,24 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, BrowserRouter, Route } from 'react-router-dom'
+import LoginScreen from './Screens/LoginScreen';
```

Comment:
This causes error. Remove this if file is not yet created.

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/70557829/176356373-80f93735-b0bb-499f-8feb-d460be54b6df.png">